stock_request_purchase/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Stock Request Purchase",
     "summary": "Internal request for stock",
-    "version": "17.0.1.0.3",
+    "version": "18.0.1.0.0",
```

Comment:
@jcadhoc Could you put version change in migration commit ? Thanks

Reply:
Ready!